File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: "); //Ask for the height of the piramide
    
    }
    while (height < 1 || height > 8); //Check if the value is between 1 and 8
    
    for (int i = 0; i < height; i++)
    {
        
        for (int k = 1; k < height - i ; k++)
        {
            printf(" "); //Print spaces
        }
        for (int j = 0; j <= i; j++)
        {
            
            printf("#"); //Print #
        }
        printf("  ");
        for (int j = 0; j <= i; j++)
        {
            
            printf("#"); //Print #
        }
        
        printf("\n"); //New line
    }
    
    
 
  
    
} 

File: pset6/mario/mario.py
from cs50 import get_int

height = -1
while height < 1 or height > 8:
    height = get_int("Height:\n")

for i in range(height):
    j = 1
    while j < height - i:
        j += 1
        print(" ", end="")
    for k in range(i + 1):
        print("#", end="")
    print("  ", end="")
    for l in range(i + 1):
        print("#", end="")
    print()
